src/util/get-github-client-config.ts

```diff
@@ -63,13 +64,38 @@ const calculateProxyBaseUrl = async (jiraHost: string, gitHubBaseUrl: string | u
 	return envVars.PROXY;
 };
 
+const calculateApiKeyConfig = async (jiraHost: string, logger: Logger): Promise<{ apiKeyConfig: GitHubClientApiKeyConfig } | undefined> => {
+	try {
+		const maybeApiKey = await stringFlag(StringFlags.GHE_API_KEY, "", jiraHost);
+		if (maybeApiKey) {
+			logger.info("Encrypted API key found");
+			const maybeApiKeyParts = maybeApiKey.split(":");
```

Comment:
If we don't use `:` as the separator, then we don't have to `maybeApiKeyParts.join(":");` later on?
How about `maybeApiKey.split("##KEY_VALUE_SPT##");` or something similar?

Reply:
I found the code hard to read, `split(":")` and `join(":")`

Reply:
Replaced with JSON.